bl-kernel/admin/themes/booty/html/media.php

```diff
@@ -94,7 +88,7 @@ function hideMediaAlert() {
 }
 
 // Show the files in the table
-function displayFiles(files) {
+function displayFiles(files, numberOfPages = <?= $numberOfPages ?>) {
```

Comment:
The PHP by default is `<?php $numberOfPages ?>` not all the PHP servers allow that.

Reply:
From https://www.php.net/manual/en/language.basic-syntax.phptags.php It says it is enabled by default in all installations since PHP 7.4

```
5.4.0 | The tag <?= is always available regardless of the short_open_tag ini setting.
```